spec/controllers/votes_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe VotesController, type: :controller do
+
+  let(:user) { FactoryGirl.create(:user) }
+  let(:question) { FactoryGirl.create(:question, user: user) }
+  let(:answer) { FactoryGirl.create(:answer, user: user) }
+
+  describe "POST#up" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.